src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
```

Comment:
The variable 'propertyValues' is not necessary, as you can check for duplicates and create the inverse object in a single loop through the 'robot' object's keys.